scss/print.scss

```diff
@@ -23,26 +23,29 @@
   .app-layout__note-column,
   .note-editor,
   .note-detail-wrapper,
-  .note-detail {
-    display: block;
-    overflow: visible;
-  }
-
+  .note-detail,
+  .note-detail-textarea,
+  .note-content-editor-shell,
   .note-detail-wrapper {
     overflow: visible;
   }
 
-  .note-detail {
-    overflow: visible;
-  }
-
   [class^='note-detail-'] {
     max-width: 100%;
     width: 100%;
     color: $studio-black;
 
     &.theme-color-fg {
       color: $studio-black;
+      overflow-y: auto !important;
     }
   }
+
+  .note-content-print {
+    left: 0px;
+    top: 0px;
```

Comment:
if `display: none` wasn't taking this out of the layout flow maybe we need to add `visibility: hidden;`?

I don't think we should need to position it at the top left of the screen and am a bit worried this could run into unexpected consequences: maybe even the flickering that @codebykat noticed

Reply:
Removed the absolute positioning.

The flickering is due to adding the display none to and then removing it. I am not sure how we fix the flicker without removing the animations. You can open the animations panel in chrome dev tools to see what it is doing.